### DISCUSSION TITLE
Transform Jira Keys to links in commit message

### DISCUSSION THREAD

**Discussion by thimmy687** [gitextensions/gitextensions#9422] [Q&A] [answered]
## Feature description

As user I'd like to click in the commit info on a jira key and the configured Jira should open on the defined issue.

## Environment

- Git Extensions 3.5.1.12196
- Build 9099a1e76d01dbbd6896d6b026027db0b68fba96
- Git 2.21.0.windows.1 (recommended: 2.30.0 or later)
- Microsoft Windows NT 10.0.19041.0
- .NET Framework 4.8.4341.0
- DPI 96dpi (no scaling)

**Comment by vbjay** [comment, marked as answer]
Open settings. Look at the revision link settings.
